### DISCUSSION TITLE
Zeek file extraction limit

### DISCUSSION THREAD

**Discussion by EbolaWare** [Security-Onion-Solutions/securityonion#10699] [Ideas] [answered]
Digging into SecOn again, looking to improve capabilities... While googling my issue, I came across [this chain](https://marc.info/?l=security-onion&m=150025769802589&w=2) and noticed the default value used to be around 100MB. But in SecOn now: [~9MB](https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/zeek/policy/securityonion/file-extraction/extract.zeek). I came to this while tracking down another strelka error and noticed quite a few "unexpected EOF" errors.

Just curious if there was a reason behind this change I missed. Maybe somewhere in documentation? I'm getting ready to bump up that value, as well as enabling archive/zip file carving since those aren't enabled by default either. Honestly can't remember the last file I downloaded that was less than 9MB...

Hopefully I can come back with some actual links to the values in SecOn...
- eW

**Comment by TOoSmOotH** [comment, marked as answer]
The 9MB was originally set because that was the limit of several sandboxes at the time the script was written. This is a balancing act because the larger the files the longer the process time in yara. You can adjust the size to whatever fits your needs just keep that in mind.
